guides/common/modules/proc_upgrading-rhel7-to-rhel8.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Upgrading-Project-from-RHEL7-to-RHEL8_{context}"]
+= Upgrading {Project} From {RHEL} 7 to {RHEL} 8
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+
+.Prerequisites
+* {Project} {ProjectVersion} is supported on both {RHEL} 7 and {RHEL} 8.
+* Subsequent minor versions of {Project} 6 will potentially be supported on {RHEL} 8 only.
+* If you have {Project} {ProjectVersion} running on {RHEL} 7, you must migrate to {RHEL} 8 to stay up-to-date on {Project} {ProjectVersion} minor releases.
+* {Project} {ProjectVersion} running on {RHEL} 7.
```

Comment:
Please reword.

Reply:
If I understand correctly, @maximiliankolb means that this sentence doesn't have a verb.